packages/client/src/fetchMetadata.test.ts

```diff
@@ -105,6 +105,13 @@ describe("FetchMetadata", () => {
         "primaryKeyApiName": "employeeId",
         "primaryKeyType": "integer",
         "properties": {
+          "class": {
+            "description": "",
+            "displayName": undefined,
+            "multiplicity": false,
+            "nullable": true,
+            "type": "string",
+          },
```

Comment:
We were assuming this property in places but did not actually define it which caused me some problems early on.